stable/contour/templates/clusterrole.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.rbac.create -}}
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Instead of the beta version of the RBAC API can you please use the release version. The release version has been available long enough that even unsupported versions of Kubernetes support it. This applies to all the RBAC objects.

Reply:
Changed all RBAC API version to release version.